_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -49,90 +52,289 @@ GET _remotestore/stats/<index_name>
-|`upload_latency_in_bytes_per_sec.moving_avg`	|The average speed of remote store uploads (in bytes per second) for the last _N_ uploads. _N_ is defined in `remote_store.segment.pressure.upload_bytes_per_sec_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).		|
-|`remote_refresh_latency_in_millis.moving_avg`	|The average time taken by a single remote refresh during the last _N_ remote refreshes. _N_ is defined in `remote_store.segment.pressure.upload_time_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).	|
+| `last_successful_upload_timestamp`| The timestamp, in milliseconds, since the last successful translog file uploaded to remore-backed storage. |
+| `total_uploads.started` | The total number of attempted translog upload syncs to remote-backed storage. |
+| `total_uploads.failed` | The total number of failed translog upload syncs to remote-backed storage.   |
+| `total_uploads.succeeded` | The total number of successful translog upload syncs to remote-backed storage.  |
+| `total_upload_size.started_bytes` | The total number of bytes of translog files actively being downloaded from remote-backed storage. |
+| `total_upload_size.succeeded_bytes` | The total number of bytes of translog files successfully uploaded to remote-backed storage. |
+|`total_upload_size.failed_bytes` | The total number of bytes of translog files that failed to uploaded to remote-backed storage. |
+| `total_upload_time_in_millis` | The total time spent, in milliseconds, uploading translog files to remote-backed storage. |
+| `upload_size_in_bytes.moving_avg` | The average size of translog data, in bytes, uploaded in the last *N* downloads. *N* is defined in the `remote_store.moving_average_window_size` setting. |
+| `upload_speed_in_bytes_per_sec.moving_avg` | The average speed of translog uploads, in bytes per second, for the last *N* uploads. *N* is defined in the `remote_store.moving_average_window_size` setting.    |
+| `upload_time_in_millis.moving_avg` | The average time taken by a single translog upload, in milliseconds, since the last *N* uploads. *N* is defined in `remote_store.moving_average_window_size` setting.    |
+
+The `translog.download` object contains the following fields:
+
+|Field	|Description	|
+|:---	|:---	|
+| `last_successful_download_timestamp` | The timestamp, in milliseconds, since the last successful translog file uploaded to remote-backed storage. |
```

Comment:
```suggestion
| `last_successful_download_timestamp` | The timestamp, in milliseconds, since the last translog file successfully uploaded to remote-backed storage. |
```